### PR TITLE
Add Java EE Security JSR 375 DatabaseIdentityStore Example

### PR DESCRIPTION
**Pull request by vinayvishal**
# built_in_db_identity_store
Example demonstrating usage of `DatabaseIdentityStore` for authentication when a request is made to this application. The default datasource bundled with Glassfish has been used to map to `DatabaseIdentityStore` to be used as `IdentityStore`.

For details, please refer to project's README.

### CODE REVIEW

File: ws/javaee8/security/jsr375/built-in-db-identity-store/pom.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    General Public License Version 2 only ("GPL") or the Common Development
+    and Distribution License("CDDL") (collectively, the "License").  You
+    may not use this file except in compliance with the License.  You can
+    obtain a copy of the License at
+    https://oss.oracle.com/licenses/CDDL+GPL-1.1
+    or LICENSE.txt.  See the License for the specific
+    language governing permissions and limitations under the License.
+
+    When distributing the software, include this License Header Notice in each
+    file and include the License file at packager/legal/LICENSE.txt.
```
Comment (yaminikb) [inline ws/javaee8/security/jsr375/built-in-db-identity-store/pom.xml]: It should be just LICENSE.txt, remove path preceding it.

Comment (vinayvishal) [reply ws/javaee8/security/jsr375/built-in-db-identity-store/pom.xml]: Fixed it. In other examples too in glassfish-samples, its with the path, we probably need to modify those also.

File: ws/javaee8/pom.xml [hunk 2]
```diff
@@ -182,6 +183,12 @@
                 <version>7.0</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
+              <groupId>javax.security.enterprise</groupId>
+              <artifactId>javax.security.enterprise-api</artifactId>
+              <version>1.0-b11</version>
```
[the remark this answers is not in the document]
Comment (vinayvishal) [reply ws/javaee8/pom.xml]: Done.

File: ws/javaee8/security/jsr375/built-in-db-identity-store/pom.xml (hunk 1 above)
Comment (yaminikb) [reply ws/javaee8/security/jsr375/built-in-db-identity-store/pom.xml]: There are 2 more places you need to fix, please check.

**Pull request merged**